rust/protocol/src/storage/traits.rs

```diff
@@ -94,8 +145,10 @@ pub trait SessionStore {
     ) -> Result<()>;
 }
 
+/// Interface for storing sender key records for other users, allowing multiple keys per user.
```

Comment:
We actually use this to store our own records too, using our own address as the `sender`.

Reply:
Thanks! I decided to just remove "for other users" instead of explicitly saying it also stores our own records, since that seemed redundant.